map/jag_cycle_shift.json

```diff
@@ -1,6 +1,7 @@
 {
     "name":"Jaguar cycle shift",
     "desc":"Workaround for Jaguar games with broken mapping.",
+    "console":"Jag",
```

Comment:
Name should match https://github.com/darthcloud/BlueRetroWebCfg/blob/c7d8bea9fc5d80bc78268e69dd6e0a01e7de705d/blueretro.js#L168

I think this is only non-matching one, use Jaguar here

Reply:
done